### ISSUE TITLE
Make it work with TinyMCE 4.x

### ISSUE THREAD

**Issue by MoppieMop** [moonwave99/TinyMCELatexPlugin#1]
That would be awesome!

**Comment by benputman** [comment]
I agree! Would like to see this working for TinyMCE 4. I'm interested in helping out.

**Comment by moonwave99** [comment]
Hi guys, sorry for coming back to you this late, I am not maintaining this plugin anymore :/ But I am glad to receive a PR in order to update it!
